src/Momento.Sdk/Internal/Utils.cs

```diff
@@ -3,7 +3,10 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+
+#if !BUILD_FOR_UNITY
 using Momento.Sdk.Exceptions;
+#endif
```

Comment:
Same comment for this file - if it seems harmless to leave this intact for all builds I think it would be nice